extensions/types/vscode-mssql.d.ts

```diff
@@ -437,6 +449,38 @@ declare module 'vscode-mssql' {
 		 * Returns an access token for given user and tenant
 		 */
 		getAccountSecurityToken(account: IAccount, tenantId: string | undefined): Promise<Token>;
+
+		/**
+		 * Returns Azure subscriptions with tenant and token for each given account
+		 */
+		getAccountSessions(account: IAccount): Promise<IAzureAccountSession[]>;
+	}
+
+	export interface IAzureResourceService {
```

Comment:
Please make separate namespaces - I'd suggest a top level `azure` with separate `resource`, `sql` and `subscription` namespaces grouped within it.

Reply:
done

Reply:
@Charles-Gagnon here's the PR for vscode-mssql https://github.com/microsoft/vscode-mssql/pull/17328